### PR TITLE
Feature/as 28 appeals data panel

### PR DESCRIPTION
**Pull request by davehlewis**
## Ticket Number

AS-28

## Description of change
- Adding sidebar panel to show appeal details.
- Routing has been changed in application to include passing an appeal ID.
- Wrapper for Appeal API has been added as well as middleware to pull Appeal details.
- Sidebar will not show if no appeal present (invalid ID or API unreachable)

## Checklist

- [ ] Requires infrastructure changes
- [ ] If adding or remove environment variables (e.g. in `docker-compose.yaml`) then I have updated the appropriate Helm chart
- [ ] I have updated the documentation accordingly
- [X] My commit history in this PR is linear
- [X] New features have tests
- [ ] Breaking change (team conversation required)

## Important

Please do not merge from `master` (please only [rebase](https://github.com/foundry4/appeal-planning-decision/wiki/An-intro-to-Git-Rebase)). This keeps the history linear and easier to debug.

### CODE REVIEW

File: data/data/appeals-service-api/001-appeals.json [hunk 1]
```diff
@@ -1 +1,99 @@
-[]
+[
```
Comment (swdpcomputing) [inline data/data/appeals-service-api/001-appeals.json:1]: Is this file part of the appeals data panel? Should it be part of a wider story on setting up data?

Comment (davehlewis) [reply data/data/appeals-service-api/001-appeals.json:1]: This is the first instance of using the data so needs the setup. Same as the other stories - the setup has been made part of the story

File: packages/lpa-questionnaire-web-app/src/middleware/fetch-appeal.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
+const { getAppeal } = require('../lib/appeals-api-wrapper');
+
+/**
+ * Middleware to ensure any ppeal data is populated
```
Comment (swdpcomputing) [inline packages/lpa-questionnaire-web-app/src/middleware/fetch-appeal.js:4]: Typo

File: data/data/appeals-service-api/001-appeals.json (hunk 1 above)
Comment (MrSimonEmms) [reply data/data/appeals-service-api/001-appeals.json:1]: We can always add more pieces of seed data as needed - I would be surprised if this were the only piece of seed data we ever need

**Review by swdpcomputing — APPROVED** [review]

**Pull request merged**